private/buf/cmd/buf/buf_test.go

```diff
@@ -1027,6 +1028,518 @@ func TestCheckLsBreakingRulesFromConfigExceptDeprecated(t *testing.T) {
 	}
 }
 
+func TestLsModulesWorkspaceV1(t *testing.T) {
+	// Cannot be parallel since we chdir.
+	pwd, err := osext.Getwd()
+	require.NoError(t, err)
+	defer func() {
+		r := recover()
+		assert.NoError(t, osext.Chdir(pwd))
+		if r != nil {
+			panic(r)
+		}
+	}()
+
+	require.NoError(t, osext.Chdir(filepath.Join(pwd, "testdata", "lsmodules", "workspacev1")))
```

Comment:
It could be handy if `buf config ls-modules` accepted an optional arg `<dir>` so you could use it without relying on chdir.

Reply:
I think `--config` does that, except it doesn't accept a `buf.work.yaml`